docs/installation.rst

```diff
@@ -163,6 +163,23 @@ FUSE for OS X, which is available as a pre-release_.
 
 .. _pre-release: https://github.com/osxfuse/osxfuse/releases
 
+
+FreeBSD
+++++++++
+
+If you're using FreeBSD, you will need to install a package and load the fuse kernel module:
+
+::
+
+    $ sudo pkg install fusefs-libs
+    $ sudo kldload fuse
+    $ borg mount /mnt/backup::root-2013-08-02 /tmp/mymountpoint
+    $ ls /tmp/mymountpoint
+    bin  boot  etc  lib  lib64  mnt  opt  root  sbin  srv  usr  var
+    $ umount /tmp/mymountpoint
+
+
+
```

Comment:
> I needed some more stuff to get it installed on freebsd:

But that's already covered here:
https://github.com/borgbackup/borg/blob/master/docs/installation.rst#dependencies

 > e.g. one does not want to kldload all the time after each reboot).

Okay, I can change that.

> Also, do not give a mount / umount example here, that is not related to installation.

fair enough.